pkgs/by-name/ko/kolla/package.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, python311Packages
+, fetchgit
+, bashate
+}:
+
+let
+  pythonPackages = python311Packages;
+
+in
+pythonPackages.buildPythonApplication rec {
+  pname = "kolla";
+  version = "18.1.0";
+
+  pyproject = true;
+
+  src = fetchgit {
+    url = "https://opendev.org/openstack/kolla";
+    hash = "sha256-jLD6ILihymQlWkkpGYC4OX8BKLpQurAK6Y5Xpju+QAI=";
+    rev = version;
+  };
```

Comment:
Can we use fetchGitea here? They look compatible at first glance. Fetching a tarball would be a lot cheaper.

Reply:
Unfortunately, their Gitea is somewhat broken:
```
--2024-08-06 15:29:45--  https://opendev.org/openstack/kolla/archive/18.1.0.tar.gz
Resolving opendev.org (opendev.org)... 38.108.68.66
Connecting to opendev.org (opendev.org)|38.108.68.66|:443... connected.
HTTP request sent, awaiting response... No data received.
```

There's also an automated mirror on Github. Do you prefer that or should we stick with `fetchgit` from the authoritative source?

Reply:
I think in other places I have used the github mirror.